### DISCUSSION TITLE
Speed Index on Performance Tab

### DISCUSSION THREAD

**Discussion by hc-keanu** [harlan-zw/unlighthouse#222] [Ideas]
When looking at the Unlighthouse report in my browser. I would love to see the the lighthouse Speed Index as a column.

Also, what do we think about the ability to download the entire report? Can this already be done and I'm just missing it?

I would love to see average totals at the bottom or top of each column. Being able to quickly see the average LCP, Speed Index or Score would be amazing.

**Comment by harlan-zw** [comment]
Thanks for the suggestion, I've pushed out v0.13.0 which includes changes to the performance columns, including the Speed Index.

All of the reports are saved locally as JSON, if you need it in a different format you can open the report via the UI and save it however you like.

![image](https://github.com/harlan-zw/unlighthouse/assets/5326365/fcd77d55-a299-4eb2-bbcb-fe0d111bdfa6)

I don't have capacity to work on the averages at this point, PRs welcome though :)
